base/iterators.jl

```diff
@@ -240,6 +241,34 @@ end
 iteratorsize{I1,I2}(::Type{Zip{I1,I2}}) = zip_iteratorsize(iteratorsize(I1),iteratorsize(I2))
 iteratoreltype{I1,I2}(::Type{Zip{I1,I2}}) = and_iteratoreltype(iteratoreltype(I1),iteratoreltype(I2))
 
+# unzip
+"""
+
```

Comment:
I would have thought that `unzip` would need it's own method name, something like `unzip(iter::AbstractZipIterator)`. Also, I think that you need to add `unzip` to `exports.jl`.